### PR TITLE
Adding the admin site

### LINKED ISSUE #99: Admin site

### PR DESCRIPTION
**Pull request by GalOved**
Adding the admin site to tasties project.
The admin site shows all the models we created and the information found in the database.
close #99 
image

### CODE REVIEW

File: tasties/urls.py [hunk 1]
```diff
@@ -13,10 +13,9 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.contrib import admin
 from django.urls import path
 from tasties_app import views
-
+from django.contrib import admin
```
Comment (GilShechter) [inline tasties/urls.py:18]: You remove this line and then adding it again

**Review by GilShechter — CHANGES_REQUESTED** [review]
Requested a small change.
Rather than that, looks okay.

File: tasties/urls.py (hunk 1 above)
Comment (GalOved) [reply tasties/urls.py]: I remove the change from the urls file

**Review by GilShechter — APPROVED** [review]

**Review by GilShechter — APPROVED** [review]
LGTM

**Review by efraimvis — APPROVED** [review]
Nice.

**Review by ohadrevach — DISMISSED** [review]

**Comment by svatares** [comment]
@GalOved please resolve conflicts

**Review by svatares — DISMISSED** [review]

**Comment by efraimvis** [comment]
@GalOved They're probably waiting for you to squash your commits.

**Review by natifridman — APPROVED** [review]
Please squash the commits

**Review by svatares — APPROVED** [review]

**Pull request merged**